add reference links to tour page

jwwicks opened the pull request:
Fixes #10849

jwwicks committed:
Link to Value Types table

jwwicks committed:
changing reference url

jwwicks committed:
Adding ref links to major points

dnfclas commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=11852) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=11852) before we can accept your contribution.<br/><br/>:x: jwwicks [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=11852)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=11852) it.</sub>

jwwicks committed:
Updating links across docsets

jwwicks committed:
changing paths

jwwicks committed:
adding one more parent to path

jwwicks committed:
fixing parens

jwwicks committed:
one less parent  without paren

jwwicks committed:
change to programming guide

jwwicks committed:
Link to Value Types table 


changing reference url


Adding ref links to major points


Updating links across docsets


changing paths


adding one more parent to path


fixing parens 


one less parent  without paren


change to programming guide

jwwicks committed:
Merge remote-tracking branch 'origin/feature/Iss-10849_AddLinksToBuiltInsPage' into feature/Iss-10849_AddLinksToBuiltInsPage

BillWagner reviewed (approved):
Thanks for making these changes @jwwicks We appreciate it!

I've reviewed the changes, and I'll :shipit: now.

Thanks again for making your first contribution to docs! 🎆
